libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -2885,18 +2886,22 @@ bool AP_AHRS::get_vel_innovations_and_variances_for_source(uint8_t source, Vecto
 //get the index of the active airspeed sensor, wrt the primary core
 uint8_t AP_AHRS::get_active_airspeed_index() const
```

Comment:
This flow is quite silly - we should probably `eliminate get_active_airspeed_index` and just ask the backends what the airspeed measurement is (if available).  Not for this PR, however....